Sources/Build/BuildPlan.swift

```diff
@@ -544,11 +544,15 @@ public final class SwiftTargetBuildDescription {
 
     /// The modulemap file for this target, if any.
     private(set) var moduleMap: AbsolutePath?
+    
+    /// The results of having applied any extensions to this target.
+    public private(set) var extensionEvaluationResults: [ExtensionEvaluationResult]
```

Comment:
does this needs to be a "var" if it is set on the constructor?

Reply:
Good point — that can be a let.  I modelled it after the others but there's no need.

Reply:
I have made this fix.